core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -420,29 +425,27 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
                     // NB: logBuilder.endResponse() is called by HttpResponseSubscriber below.
                     if (!isTransientService) {
                         gracefulShutdownSupport.dec();
-                    }
-                    unfinishedRequests.remove(req);
-                    if (unfinishedRequests.isEmpty() && handledLastRequest) {
+                    } unfinishedRequests.remove(req); if (unfinishedRequests.isEmpty() && handledLastRequest) {
```

Comment:
It looks like these lines need reformatting

Reply:
Oops... it should be reverted.